debug-config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should check in a `debug-config.json-dist` version of this, so developers' customized local copies don't collide with checked in files. Also, establish defaults in the case where this files doesn't exist.